### PR TITLE
Remove claims and classes

### PR DESCRIPTION
**Pull request by muvaf**
### Description of your changes

This PR also removes the compute/v1alpha3 EKSCluster, which we have been keeping around because the v1beta1 EKSCluster is compatible only with composition and not resource claims.

See https://github.com/crossplane/crossplane/issues/1670 for context.
 

### Checklist

I have:
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Ensured this PR contains a neat, self documenting set of commits.
- [ ] Updated any relevant [documentation], [examples], or [release notes].
- [ ] Updated the dependencies in [`app.yaml`] to include any new role permissions.

[documentation]: https://github.com/crossplane/crossplane/tree/master/docs
[examples]: https://github.com/crossplane/crossplane/tree/master/cluster/examples
[release notes]: https://github.com/crossplane/crossplane/tree/master/PendingReleaseNotes.md
[`app.yaml`]: https://github.com/crossplane/provider-aws/blob/master/config/package/manifests/app.yaml

### CODE REVIEW

**Comment by muvaf** [comment]
Updated with `master` branches and manually tested.

**Review by hasheddan — CHANGES_REQUESTED** [review]
@muvaf I believe you'll need to remove package install from integration tests here: https://github.com/crossplane/provider-aws/blob/eab4250ee41843b8f4e317facae82fe213747179/cluster/local/integration_tests.sh#L236

File: pkg/controller/s3/s3bucket.go [hunk 1]
```diff
@@ -210,7 +208,7 @@ func updateTagging(bucket *bucketv1alpha3.S3Bucket, client s3.Service) error {
 
 func (r *Reconciler) _delete(bucket *bucketv1alpha3.S3Bucket, client s3.Service) (reconcile.Result, error) {
 	bucket.Status.SetConditions(runtimev1alpha1.Deleting(), runtimev1alpha1.ReconcileSuccess())
-	if bucket.Spec.ReclaimPolicy == runtimev1alpha1.ReclaimDelete {
+	if bucket.Spec.DeletionPolicy == runtimev1alpha1.DeletionDelete {
```
Comment (negz) [inline pkg/controller/s3/s3bucket.go:211]: iirc Delete is now the default. Do we need to handle the case where the policy not set?

**Review by negz — APPROVED** [review]
I left one comment, and I think the integration tests will need to be disabled, but otherwise LGTM. For the integration tests I've taken an approach of removing them completely in a separate commit so that we can easily revert and update when the new packages are ready.

File: pkg/controller/s3/s3bucket.go (hunk 1 above)
Comment (muvaf) [reply pkg/controller/s3/s3bucket.go:211]: This controller will be [gone](https://github.com/crossplane/provider-aws/pull/331) in the next days, so I don't think we need to handle it. I just made changes to get the controller in a state that can be compiled.

**Review by hasheddan — APPROVED** [review]
I'll add back integration tests when new packaging is ready :+1:

**Pull request merged**